### PR TITLE
dist: allow specify JVM options from sysconfig

### LINKED ISSUE #58: Use external/user provided JVM_OPTS values
HEAD: df2dee2402af75a4293eb149b835d96d76c2d44e

*Description*
Add the `exec` line the ability to incorporate parameters from the external `JVM_OPTS` macro (e.g. set in the `/etc/sysconfig/scylla-jmx`).

This is useful when users try to enable tools like `jolokia` to work with out JMX.

### PR DESCRIPTION
**Pull request by syuu1228**
Add SCYLLA_JMX_JVM_OPTS on sysconfig to specify JVM options.

fixes #58

### CODE REVIEW

File: dist/common/sysconfig/scylla-jmx [hunk 1]
```diff
@@ -27,3 +27,6 @@ SCYLLA_JMX_LOCAL="-l /opt/scylladb/jmx"
 
 # allow debug
 #SCYLLA_JMX_DEBUG="-d"
+
+# specify JVM options
+SCYLLA_JMX_JVM_OPTS=""
```
Comment (tarzanek) [inline dist/common/sysconfig/scylla-jmx]: why not using JVM_OPTS directly and export them?
would save you later the hack with --

Comment (syuu1228) [reply dist/common/sysconfig/scylla-jmx]: Seems like it's JAVA_TOOL_OPTIONS not JVM_OPTS, for environment variable.

https://stackoverflow.com/questions/2011311/running-java-with-java-opts-env-variable-has-no-effect
https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars002.html

I'll send new version using the variable.

Comment (syuu1228) [reply dist/common/sysconfig/scylla-jmx]: Pushed changes.

Comment (tarzanek) [reply dist/common/sysconfig/scylla-jmx]: nice, lgtm from my side, feel free to merge

**Comment by syuu1228** [comment]
@tarzanek please do 'approve' on github.
@penberg @avikivity ping

**Comment by tarzanek** [comment]
hello, I have no rights, so it's down to @penberg or @avikivity

**Comment by penberg** [comment]
We need to get this documented somewhere, but where @tzach?

File: dist/common/sysconfig/scylla-jmx [hunk 2]
```diff
@@ -27,3 +27,6 @@ SCYLLA_JMX_LOCAL="-l /opt/scylladb/jmx"
 
 # allow debug
 #SCYLLA_JMX_DEBUG="-d"
+
+# specify JVM options
+JAVA_TOOL_OPTIONS=""
```
[the remark this answers is not in the document]
Comment (avikivity) [reply dist/common/sysconfig/scylla-jmx]: (I made this comment before @penberg committed, but github held it in pending state)

[the remark this answers is not in the document]
Comment (avikivity) [reply dist/common/sysconfig/scylla-jmx]: Anyway, it doesn't matter.

**Pull request merged**